metricbeat/values.yaml

```diff
@@ -259,6 +259,10 @@ clusterRoleRules:
     resources:
       - nodes/stats
     verbs: ["get"]
+  - apiGroups: ["coordination.k8s.io"]
+    resources:
+      - leases
+    verbs: ["create", "get", "list", "update"]
```

Comment:
I think this should also be added to https://github.com/elastic/helm-charts/blob/c52ad079379e6b1ba420866cdf96019f382f94ff/filebeat/values.yaml#L203-L213

Reply:
I guess just role binding is ok? https://github.com/elastic/beats/blob/878e778aed26ca30a660157aa2e11cd9f6a899ef/deploy/kubernetes/metricbeat-kubernetes.yaml#L315